src/com/jester/view/View.java

```diff
@@ -1,659 +1,666 @@
-                albumSongs[ctr2].setBorder(BorderFactory.createLineBorder(Color.GRAY, 2));
-                albumSongs[ctr2].setFont(new Font("Arial", Font.BOLD, 16));
-                songContainer[ctr].add(albumSongs[ctr2]);
-            }
-            albumPanels[ctr].add(albumHeader[ctr], BorderLayout.NORTH);
-            albumPanels[ctr].add(songContainer[ctr], BorderLayout.CENTER);
-            /*album panels creation end */
-            
-            cardPanel.add(albumPanels[ctr], str2);
-            albumButton[ctr].setText(str1);
-            albumButton[ctr].setFont(new Font("Arial", Font.ITALIC, 16));
-            albumButton[ctr].setBorder(BorderFactory.createLineBorder(Color.WHITE,2));
-            albumButton[ctr].setHorizontalAlignment(SwingConstants.LEFT);
-            albumButton[ctr].setBackground(new Color(118,113,113,255));
-            albumButton[ctr].setForeground(Color.WHITE);
-            albumButton[ctr].addActionListener(new ActionListener() {
-                @Override
-                public void actionPerformed(ActionEvent e) {
-                    cl.show(cardPanel, str2);
-                }
-            });
-            albumContainer.add(albumButton[ctr]);
-
-        }
-        albumViewer.add(albumContainer, BorderLayout.CENTER);
-        JScrollPane scrollableArea = new JScrollPane(albumViewer);
-        scrollableArea.setVerticalScrollBarPolicy(JScrollPane.VERTICAL_SCROLLBAR_ALWAYS);
-        cardPanel.add(scrollableArea, "4");
-
-        /*ALbum Viewer Editor End*/
-        
-        
-       
-        //Adding the cardPanel in the headerPanel
-        mainBorderPanel.add(headerPanel, BorderLayout.NORTH);
-        mainBorderPanel.add(cardPanel, BorderLayout.CENTER);
-        
-        //Adding the JPanels inside the frame
-        frame.getContentPane().add(mainBorderPanel, BorderLayout.CENTER);
-        frame.getContentPane().add(sideBar, BorderLayout.LINE_START);
-        frame.getContentPane().add(playArea, BorderLayout.PAGE_END);
-        
-        //Specifying how the window closes and displaying the window.
-        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-        frame.getContentPane().setPreferredSize(new Dimension(1200, 720));
-        frame.pack();
-        frame.setVisible(true);
-    } 
-    
-    private static void music_lib_refresher(String music_title, String music_artist, String music_album) {
-        Object[] row = new Object[50];
-        music_ls = controller.GetAllStoredMusic(); 
-        // Insert the new record to the table model ONE TIME ONLY! 
-        for(int i = 0; i < 1; i++) {
-            row[0] = music_title; 
-            row[1] = music_artist; 
-            row[2] = music_album; 
-            
-            model.addRow(row);
-        }
-    }
-    
-    private static String getTitleVal() {
-        return title_play;
-    }
-    
-    public static void main(String[] args) {
-        SwingUtilities.invokeLater(View::view);
-    }
-    
-}
+/**
```

Comment:
Please, avoid downloading and just replacing the source file in the future. Merge conflicts can happen. to : Reymaaan123